packages/sysutils/busybox/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="busybox"
-PKG_VERSION="9aa751b08ab03d6396f86c3df77937a19687981b"
-PKG_SHA256="41d6f6ad37d21bace07ec596e72c1aa0874b8ccd1d9ef9a609a07b37cabea657"
+PKG_VERSION="e16f7eb5967b9a960f4600c20690af63fb830b60"
```

Comment:
please change that to the latest release tag (1_32_0), switching to dev snapshots was only an interim solution to fix issues after glibc update and to test changes
see https://github.com/LibreELEC/LibreELEC.tv/commit/d1775f8fe0d82b450b280206718ea710a877c06f and https://github.com/LibreELEC/LibreELEC.tv/pull/4325

Reply:
Got it.